sdk/ml/azure-ai-ml/tests/component/unittests/test_command_component_entity.py

```diff
@@ -87,6 +101,34 @@ def test_command_component_entity(self):
 
         assert component_dict == yaml_component_dict
 
+    def test_command_component_with_additional_includes(self):
+        tests_root_dir = Path(__file__).parent.parent.parent
+        samples_dir = tests_root_dir / "test_configs/components/"
+
+        component = CommandComponent(
+            name="additional_files",
+            description="A sample to demonstrate component with additional files",
+            version="0.0.1",
+            command="echo Hello World",
+            environment=Environment(image="zzn2/azureml_sdk"),
+            # as sdk will take working directory as root folder, so we need to specify the absolution path
+            additional_includes=[
```

Comment:
What happens when code not specified and additional_includes is an empty list?

Reply:
will not upload anything and code is None